### ISSUE TITLE
Using `strip` with `include-regex` is breaking build

### ISSUE THREAD

**Issue by mooyoul** [prebuild/prebuild#289]
Hello all,

I just found there is an edge case scenario that breaks the `strip` command.

For example, ​`.prebuildrc` is configured like below:
```json
{
  "target": ["14.0.0","16.0.0"],
  "include-regex": "\\.(node|a|so|dylib)",
  "strip": true,
  "prepack": "./build/prepack.sh"
}
```

and `build/preback.sh` 

```sh
#!/usr/bin/env bash

set -e

# Cleanup
rm -f build/Release/*.so
readarray -t COPY_REQUIRED_LIBS < <( lddtree -l build/Release/my-package.node | sed -r -e '/^\/lib/d' -e '/my-package.node/d' )

for LIB_SO_FILENAME in "${COPY_REQUIRED_LIBS[@]}"; do
  LIB_SO_SRC="vendor/my-package/usr/local/lib/${LIB_SO_FILENAME}"
  LIB_SO_DEST="build/Release/$(basename "$LIB_SO_SRC")"
  echo "Copying $LIB_SO_SRC to $LIB_SO_DEST"
  cp "$LIB_SO_SRC" "$LIB_SO_DEST"
  patchelf --set-rpath '$ORIGIN' "$LIB_SO_DEST"
done;

ls -al build/Release
```

In this case, `collectArtifacts` yields not only `my-package.node` but also copied libs (e.g. libfoo.so). and those filenames passed to stripArgs, This leads to invalid strip args (comma-separated list).

I think this is a similar issue with #246.

**Comment by mooyoul** [comment]
Oh I just saw #280 and this PR looks good to me. @vweevers Can you check that? Thanks.

**Comment by vweevers** [comment]
@mooyoul Does #280 (released in 11.0.3) solve your issue?

**Comment by mooyoul** [comment]
@vweevers Just updated to v11.0.3 and tested - Can confirm #280 fixes this issue 🎉  Thank you for taking your time!
